rb/lib/selenium/webdriver/devtools/response.rb

```diff
@@ -35,7 +35,7 @@ def self.from(id, encoded_body, params)
             id: id,
             code: params['responseStatusCode'],
             body: (Base64.strict_decode64(encoded_body) if encoded_body),
-            headers: params['responseHeaders'].each_with_object({}) do |header, hash|
+            headers: (params['responseHeaders'] ||  []).each_with_object({}) do |header, hash|
```

Comment:
Interestingly, if we'd used `#fetch` here we would have had a better error message. That was your initial suggestion, any reason for the change? We probably should be moving in that direction for our Hash implementations in general (tough to enforce that in a linter)

Reply:
Done. Zero Rubocop warning and used `params.fetch`

Reply:
Thanks, sorry for delay